mne_bids/copyfiles.py

```diff
@@ -153,6 +154,77 @@ def copyfile_ctf(src, dest):
                   op.join(dest, bids_folder_name + ext))
 
 
+def copyfile_kit(src, dest, subject_id, session_id,
+                 task, run, _init_kwargs, make_bids_basename):
+    """Copy and rename KIT files to a new location.
+
+    Parameters
+    ----------
+    src : str
+        Path to the source raw .con or .sqd folder.
+    dest : str
+        Path to the destination of the new bids folder.
+    subject_id : str | None
+        The subject ID. Corresponds to "sub".
+    session_id : str | None
+        The session identifier. Corresponds to "ses".
+    task : str | None
+        The task identifier. Corresponds to "task".
+    run : int | None
+        The run number. Corresponds to "run".
+    _init_kwargs : dict
+        Extract information of marker and headpoints
+    make_bids_basename : helper function
```

Comment:
This function is already imported above, you don't need to pass it again.

Reply:
No, it didn't. And I got import error when importing `make_bids_basename` from `mne_bids.write` and `copyfile_kit` is called when executing `write_raw_bids`. Thus why I pass it rather than import `make_bids_basename` at the beginning.

Reply:
Okay but just import it instead of passing it?

Reply:
There will be an import error when `copyfile_kit` is called inside `write_raw_bids` where `make_bids_basename` is defined.

Reply:
uggh, it's probably related to `https://github.com/mne-tools/mne-bids/issues/299`. We should move this function into utils.py. For now, can you do a "nested import". You can import it inside the function and it should not throw the import error

Reply:
thanks for the trick!